build_docker.sh

```diff
@@ -0,0 +1,2 @@
+docker buildx build -f docker/Dockerfile --target iroh --platform linux/arm64/v8 --tag n0computer/iroh:latest .
+# docker buildx build -f docker/Dockerfile --target iroh --platform linux/amd64 --tag n0computer/iroh:latest .
```

Comment:
what about this?

Reply:
We should drop this, stuff is already there in the readme.